api/src/TestHelpers/NamedDeputyTestHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+
```

Comment:
I know it's minor but should this be one space here rather than 2?

Reply:
You know, for about 3 years I've assumed it was PSR-12 to leave two lines between <?php and the namespace as that's what PHPStorm gives you but its totally not in there! I'll remove 😅

Reply:
I've just added PSR-12 to the pre-commit hook for this branch too so there should be no more cases of formatting PR comments going forward 🙏